### PR TITLE
Add ALMA project metadata query tool

### PR DESCRIPTION
**Pull request by keflavich**
This is a very simple tool with no parsing, so it doesn't get a local test, only a remote one

### CODE REVIEW

**Comment by equant** [comment]
In the documentation/comment of the function get_project_metadata() you have "the project aspect".  I'm not familiar with "aspect".  Should this be changed to "abstract" or at least have "abstract" added to the description?

File: astroquery/alma/core.py [hunk 1]
```diff
@@ -1055,6 +1055,21 @@ def _json_summary_to_table(self, data, base_url):
         tbl = Table([Column(name=k, data=v) for k, v in iteritems(columns)])
         return tbl
 
+    def get_project_metadata(self, projectid, cache=True):
+        """
+        Get the metadata - specifically, the project abstract - for a given project ID.
+        """
+        url = urljoin(self._get_dataarchive_url(), 'aq/')
+
+        assert len(projectid) == 14, "Wrong length for project ID"
```
Comment (bsipocz) [inline astroquery/alma/core.py]: I'm not particularly excited about these asserts in code, but I suppose it'll do for now and we can clean up later (famous last words).

**Comment by bsipocz** [comment]
I go ahead and merge this now. Skipped CI as it only adds a remote test, that has passed locally.

**Comment by bsipocz** [comment]
thanks @keflavich!

**Pull request merged**